README.md

```diff
@@ -95,14 +95,20 @@ Gatsby is a modern web framework for blazing fast websites.
 
 You can get a new Gatsby site up and running on your local dev environment in 5 minutes with these four steps:
 
-1. **Install the Gatsby CLI.**
+1. Install Node.js
+
+In order to execute the Gatsby CLI (command-line interface) you will need to have [Node.js](https://nodejs.org/) installed on your machine. Node.js comes with `npm` and `npx` pre-installed.
```

Comment:
I think a better approach would be to add a note about Node being a prerequisite, and centralize the install instructions on a single doc on gatsbyjs.org so we can keep it updated. Here's an issue related to that: https://github.com/gatsbyjs/gatsby/issues/15529